force_wfmanager/left_side_pane/workflow_settings.py

```diff
@@ -90,77 +90,95 @@ def delete_entity_handler(self, editor, object):
     action='handler.delete_entity_handler(editor, object)'
 )
 
+
+class WorkflowElementNode(TreeNode):
+    """ Custom TreeNode class for worklow elements """
+    def get_icon(self, object, is_expanded):
+        if object.valid:
```

Comment:
you can write `return 'icons/valid.png' if object.valid else 'icons/invalid.png'`